content/foundations/primitives/migrating.mdx

```diff
@@ -1,10 +1,240 @@
 ---
 title: Migrating to CSS variables
 description:
-  Support for migrating from SCSS varibles to CSS variables
+  Quick references for migrating from old Primer styling systems to new CSS variable primitives
 ---
 
-## CSS variable replacements for SCSS variables
+## Migrating from `sx` props (Primer React)
+
+### Spacing
+
+> [!CAUTION]
+> If a the only available primitive is a `--base-size` primitive, this is likely not a good value for that spacing use case. For now you could use the plain pixel or `rem` value but ideally you'd adjust the design or styling approach. `--base-size` is primarily for dimensions.
+
+#### Stacks
+
+[Stacks](https://primer.style/foundations/primitives/size#stack) are boxes containing flow content; typically vertical. These are usually `display: flex` or `grid` or `block`.
```

Comment:
They are typically vertical, right? Otherwise it's hard to find the line between `control-stack` and regular `stack` values when migrating. I can't think of very many horizontal stack cases.

Reply:
ah okay, cool :+1:

Reply:
```suggestion
[Stacks](https://primer.style/foundations/primitives/size#stack) are boxes containing flow content in a vertical or horizontal layout. These are usually `display: flex` or `grid` or `block`.
```